apps/mark-scan/frontend/src/pages/ballot_successfully_cast_page.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  FullScreenIconWrapper,
+  H1,
+  Icons,
+  P,
+  appStrings,
+} from '@votingworks/ui';
+import styled from 'styled-components';
+import { DisplaySettingsButton } from '@votingworks/mark-flow-ui';
+import { CenteredPageLayout } from '../components/centered_page_layout';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin-left: 1rem;
+`;
+
+export function BallotSuccessfullyCastPage(): JSX.Element {
+  const settingsButton = <DisplaySettingsButton />;
+  return (
+    <CenteredPageLayout voterFacing buttons={settingsButton}>
+      <Container>
+        <FullScreenIconWrapper>
+          <Icons.Done color="success" />
+        </FullScreenIconWrapper>
+        <TextContainer>
+          <H1>{appStrings.titleBallotSuccessfullyCastPage()}</H1>
+          <P>{appStrings.noteThankYouForVoting()}</P>
+        </TextContainer>
```

Comment:
The icon looks too big compared to the text in the horizontal layout. Did you try vertically stacking them?

Reply:
That's reasonable. I'll update to vertically stacked

Reply:
![Screenshot 2024-01-25 at 9 26 49 AM](https://github.com/votingworks/vxsuite/assets/1060688/e3bd5867-1722-46ca-9528-3e9083d79b12)